lambdas/layers/util/util/s3.py

```diff
@@ -223,8 +229,30 @@ def upload_s3_object_local_file(bucket_name: str, local_file: str, s3_key_destin
     return res
 
 
-def upload_s3_object_from_string(bucket_name: str, byte_of_string: str, s3_key_destination: str):
-    s3_client = util.get_client('s3')
-    res = s3_client.put_object(Body=byte_of_string, Bucket=bucket_name, Key=s3_key_destination)
+def upload_s3_object_from_string(
+    bucket_name: str, byte_of_string: str, s3_key_destination: str
+):
+    s3_client = util.get_client("s3")
+    res = s3_client.put_object(
+        Body=byte_of_string, Bucket=bucket_name, Key=s3_key_destination
+    )
+
+    return res
+
+
+def get_s3_location(bucket_name: str) -> str:
+    """
+    Will return s3 bucket region
+    """
+    response = S3_CLIENT.get_bucket_location(Bucket=bucket_name)
+    return response["LocationConstraint"]
+
+
+def create_s3_sync_command_from_s3_uri(
+    source_s3_uri: str, destination_s3_uri: str
+) -> str:
+    filename = get_s3_filename_from_s3_key(source_s3_uri)
 
-    return res
\ No newline at end of file
+    source_s3_uri = source_s3_uri.strip(filename)
+    target_s3_uri = destination_s3_uri.strip(filename)
+    return f"s3 sync {source_s3_uri} {target_s3_uri} --exclude='*' --include='*/{filename}'"
```

Comment:
Hm... I see the `sync` more useful when transferring whole directories, but I guess it could be used like that...

Also, careful about the usage of `.strip()`:
`"s3://foo/test/test.txt".strip("test.txt")`  => `'3://foo/test/'`

I'd suggest `removesuffix`, but that's only available from Python 3.9... 
You could also strip/shorting the string to everything up to the last `/`.

Reply:
> Hm... I see the sync more useful when transferring whole directories, but I guess it could be used like that...

Yes the only advantage of this would be skipping if file has been copied before. Otherwise `cp` seems to fit more here. On the second thought, maybe we should change this back using `cp` and reserve the `sync` when we need to share per directory level?

> Also, careful about the usage of .strip():
"s3://foo/test/test.txt".strip("test.txt") => '3://foo/test/'

Ahh, wasn't expecting this behaviour. Thanks for raising this up.

Reply:
> Yes the only advantage of this would be skipping if file has been copied before. Otherwise cp seems to fit more here. On the second thought, maybe we should change this back using cp and reserve the sync when we need to share per directory level?

Happy either way. For now we don't know the usage patterns... do we often resubmit? Do we need folder approaches? ... We can adjust over time.